e2e/testenv/template.go

```diff
@@ -39,7 +41,7 @@ func ApplyTemplate(k kubectl.Command, asset string, data interface{}) error {
 	tmpdir, _ := os.MkdirTemp("", "fleet-")
 	defer os.RemoveAll(tmpdir)
 
-	output := path.Join(tmpdir, RandomFilename(asset))
+	output := path.Join(tmpdir, RandomFilename(asset, rand.New(rand.NewSource(time.Now().UnixNano())))) // nolint:gosec // test code
```

Comment:
We probably don't care about this filename and don't need to control the seed, as it's just an intenral temp file.